### PR TITLE
Luis Hurtado

### PR DESCRIPTION
**Pull request by luishurtado-globant**
Tarea #2 Form

### CODE REVIEW

File: luis-hurtado/tarea#2/index.html [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+                <article>
+                    <h4>Choose your favorite color:</h4>                
+                    <input type="color">
+                </article>
+                <article>
+                    <h4>Check all the applications that you have:</h4>
+                    <section>
+                        <input type="checkbox">
+                        <figure>
+                            <img src="/images/facebook.png" alt="Facebook Icon" width="70px">
```
Comment (emolinaglobant) [inline luis-hurtado/tarea#2/index.html]: @luishurtado-globant please use relative paths for all the images,

```suggestion
                            <img src="images/facebook.png" alt="Facebook Icon" width="70px">
```

File: luis-hurtado/tarea-1/index.html [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Tarea-1</title>
+</head>
+<body>
+    <main class="container" >
+        <article>
```
Comment (emolinaglobant) [inline luis-hurtado/tarea-1/index.html]: @luishurtado-globant , this should be a header

File: luis-hurtado/tarea-1/index.html [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+            <h2 id="title">VOGUE</h2>
+            <h3 id="subtitle">The rebirth of style</h3>  
+        </article>
+        <section class="description">
```
Comment (emolinaglobant) [inline luis-hurtado/tarea-1/index.html]: @luishurtado-globant , please add an article tag to enclose sections

File: luis-hurtado/tarea-1/index.html (hunk 2 above)
Comment (luishurtado-globant) [reply luis-hurtado/tarea-1/index.html]: Hi, the main tag or the article?

Comment (emolinaglobant) [reply luis-hurtado/tarea-1/index.html]: @luishurtado-globant , the article

**Comment by emolinaglobant** [comment]
@luishurtado-globant , you need to adapt your CSS styles to work properly with semantic tag
Screen Shot 2022-01-24 at 10 46 38 AM
s

**Review by emolinaglobant — APPROVED** [review]

**Pull request merged**